build.gradle

```diff
@@ -461,46 +455,49 @@ project(':iceberg-aws') {
     api project(':iceberg-api')
     implementation project(':iceberg-common')
     implementation project(':iceberg-core')
-    annotationProcessor "org.immutables:value"
-    compileOnly "org.immutables:value"
-    implementation "com.github.ben-manes.caffeine:caffeine"
+    annotationProcessor libs.immutables.value
+    compileOnly libs.immutables.value
+    implementation libs.caffeine
+    implementation platform(libs.jackson.bom)
```

Comment:
same as above

Reply:
This approach using bom was suggested by @snazy in https://github.com/apache/iceberg/pull/7694#discussion_r1227938340 & https://github.com/apache/iceberg/pull/7694#discussion_r1227943762
which actually has its own advantages